pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -132,3 +133,16 @@ def makevector(*x):
         return torch.tensor(x, dtype=torch_dtype)
 
     return makevector
+
+
+@pytorch_funcify.register(IfElse)
+def pytorch_funcify_IfElse(op, **kwargs):
+    n_outs = op.n_outs
+
+    def ifelse(cond, *true_and_false, n_outs=n_outs):
+        if cond:
+            return torch.stack(true_and_false[:n_outs])
```

Comment:
As it stands the output formatter will always call .cpu on the result, so this just ensures we have a tensor coming out.